packages/angular/cli/commands/build-long.md

```diff
@@ -1,3 +1,14 @@
+Uses the Webpack build tool, with default configuration options specified in the workspace configuration file, `angular.json`, or with a named alternative configuration. 
```

Comment:
Edit: Looks like "webpack" uses lowercase for themselves: https://webpack.js.org/concepts/